apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -220,7 +220,7 @@ export default class AssignmentSelector extends Component {
 
     const courseOfferingsByCategories = getCourseOfferingsByCategory(
       courseOfferings,
-      section.participantType
+      'student'
```

Comment:
A lot needs to be fixed from this but this is probably the trickiest but also one of the most critical